jetty-start/src/main/java/org/eclipse/jetty/start/AllOSSLicenses.java

```diff
@@ -0,0 +1,536 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.start;
+
+import java.util.AbstractMap;
+import java.util.Map;
+
+public class AllOSSLicenses
```

Comment:
I have an issue with this file.  Firstly it will need to be maintained, but more importantly it is from a generated list that says we need to see the sources to determine the licence for the list itself.   I'm not sure we can check this in and say it is EPL/Apache2

I think we should just report the raw license info we find.... perhaps there is an option to lookup this info, but do so online without checking this is.

Reply:
I don't like this file either.
Please remove it.

Reply:
But the "license" we find in `Bundle-License` is a URL like `https://www.eclipse.org/legal/epl-2.0`, so we can't really print the URL, hence this mapping file.

If we remove it, what do we report, the URLs?

Reply:
A Bundle-License can be any text. (even non-urls)
Reporting the URLs does make sense, if we have report all of the discovered License information.

Reply:
Keep in mind, that with JEP-238, it is possible to have multiple META-INF/MANIFEST.MF files too.
And they could even not agree with each other (we've seen this)

Reply:
> If we remove it, what do we report, the URLs?

The URLs are mostly humanly readable.  It is clear what `https://www.eclipse.org/legal/epl-2.0` is.
But if there is need, an option can be provided to download the source json or XML and pretty print it from that. The data is in github, perhaps it is also in maven, in which case we can even cache it locally.